README.rst

```diff
@@ -1,76 +1,51 @@
-|Build Status| |Codecov| |license|
+|Build Status|
 
-frontend-template-application
-=================================
+frontend-app-skills
+==============================
 
-Please tag **@edx/fedx-team** on any PRs or issues.  Thanks.
+Please tag **@openedx/2u-aperture** on any PRs or issues.
```

Comment:
This should be the edx github team not the openedx one

Reply:
Defer to Ned if I'm wrong about this

Reply:
The choices are `openedx/2U-aperture` or `edx/edx-aperture`, since those are the two teams that exist.